js/tic_tac_toe.js

```diff
@@ -1,2 +1,168 @@
-// Your code here
 // Use the jQuery
+var App = (function() {
+
+  var $boxCell, $box1, $box2, $box3, $box4, $box5, $box6, $box7, $box8, $box9;
+  var x = 1;
+
+  var init = function() {
+
+    $boxCell = $('.box_cell');
+    $('tr').on('click', '.box_cell', App.changeColor );
+    $('tr').on('click', '.box_cell', App.winner );
+
+  }
+
+  var changeColor = function(){
+    if($(this).attr('class') != "box_cell red" && $(this).attr('class') != "box_cell black"){
+      if (x%2 != 0) {
+        $(this).css('background-color', 'red');
+        $(this).addClass("red");
+        x += 1;
+      } else if (x%2 == 0) {
+        $(this).css('background-color', 'black');
+        $(this).addClass("black");
+        x += 1;
+      }
+    }
+  };
+
+  var winner = function(){
```

Comment:
This is definitely the long way, but it really isn't bad.. just very verbose. You'll see a few ways of checking the rows, columns and diagonals if you check out some other student's solutions. Tys has a good one for that.